src/components/GenreTag/GenreTag.tsx

```diff
@@ -2,11 +2,23 @@ import styles from "./GenreTag.module.css";
 import "../../globals.css";
 
 type GenreTagProps = {
-  text: string;
+  tag: {
+    text: string;
+    selected: boolean;
+    id: number;
+  };
+  handleClick: (value: number) => void;
 };
 
-function GenreTag({ text }: GenreTagProps): JSX.Element {
-  return <button className={styles.tag}>{text}</button>;
+function GenreTag({ tag, handleClick }: GenreTagProps): JSX.Element {
+  return (
+    <button
+      className={tag.selected ? styles.tag_selected : styles.tag_unselected}
```

Comment:
If you like BEM, it would look like this:

```suggestion
      className={`${styles.tag} ${tag.selected ? styles.tag_selected : styles.tag_unselected}`}
```

Reply:
This is great! thanks!